www/annotationdb/annotationdb.css

```diff
@@ -0,0 +1,443 @@
+div.wy-nav-content{
```

Comment:
I can take it out, but some of the page elements were pretty limited by horizontal space, so there may be parts that don't look as nice (particularly longer names in the query builder tree).